### DISCUSSION TITLE
CUSTOMIZABLE SMS LOGIN FEILDS

### DISCUSSION THREAD

**Discussion by ubansal40** [inspireui/support#5672] [New Feature/Ideas Requests]
After the otp has been verified .The register feilds must be customizable in fluxbuilder. so that we can remove confirm password ,auto set username as contact number . This will make it very easy for users to use the app.

**Comment by osman-kocak** [comment]
On deful sms login field user can login in with google or other social channels

**Comment by tryabp** [comment]
[Please upvote this](https://github.com/inspireui/support/discussions/5663). Digits SMS login with one click login/signup add-on will solve this problem and give best login and signup experience to users. 

https://github.com/inspireui/support/discussions/5663
